test/extensions/extensions_build_system.bzl

```diff
@@ -1,62 +1,47 @@
 load("//bazel:envoy_build_system.bzl", "envoy_benchmark_test", "envoy_cc_benchmark_binary", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_binary", "envoy_cc_test_library")
-load("@envoy_build_config//:extensions_build_config.bzl", "DISABLED_BY_DEFAULT_EXTENSIONS", "EXTENSIONS")
+load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
 
-def extension_allowed_for_test(extension_name):
-    return extension_name in EXTENSIONS or extension_name in DISABLED_BY_DEFAULT_EXTENSIONS
+def _skip_extension(func, name, extension_name, **kwargs):
```

Comment:
This name is a bit confusing, maybe `apply_to_extension_allow_for_test` or something.